libs/ledgerjs/script/crypto-assets-importer/importers/erc20-signature.ts

```diff
@@ -0,0 +1,22 @@
+import { log } from "console";
+import fs from "fs";
+import path from "path";
+import { fetchTokens } from "../fetch";
+
+export const importERC20Signatures = async (outputDir: string) => {
+  log("importing erc20 signatures...");
+  try {
+    const erc20Signatures = await fetchTokens<string>("erc20-signatures.json");
+    const filePath = path.join(outputDir, "erc20-signatures");
+    if (erc20Signatures) {
+      fs.writeFileSync(`${filePath}.json`, JSON.stringify(erc20Signatures));
+      fs.writeFileSync(
+        `${filePath}.ts`,
+        `export default ${JSON.stringify(erc20Signatures, null, 2)};`,
```

Comment:
We could unit test with mock fetchTokens outputs and importERC20Signatures, applicable for every methods